.kitchen.yml

```diff
@@ -4,6 +4,9 @@ driver:
   customize:
     memory: 2048
     cpus: 2
+  # TESTING: Uncomment synced_folders to test local mixlib-install builds
+  # synced_folders:
+  #   - ["/Users/patrickwright/github/mixlib-install/pkg", "/pkg"]
```

Comment:
We might want to more generic:

```
#   - ["~/github/mixlib-install/pkg", "/pkg"]
```

We might also want to create a `.kitchen.mixlib-install.yml` file which could be loaded via the `KITCHEN_LOCAL_YAML` env var.